backend/src/main/java/c/team/message/MessageConfiguration.java

```diff
@@ -0,0 +1,23 @@
+package c.team.message;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.messaging.simp.config.MessageBrokerRegistry;
+import org.springframework.web.socket.config.annotation.EnableWebSocketMessageBroker;
+import org.springframework.web.socket.config.annotation.StompEndpointRegistry;
+import org.springframework.web.socket.config.annotation.WebSocketMessageBrokerConfigurer;
+
+@Configuration
+@EnableWebSocketMessageBroker
+public class MessageConfiguration implements WebSocketMessageBrokerConfigurer {
+
+    @Override
+    public void registerStompEndpoints(final StompEndpointRegistry registry){
+        registry.addEndpoint("/session-handling").withSockJS();
+    }
+
+    @Override
+    public void configureMessageBroker(final MessageBrokerRegistry registry) {
+        registry.setApplicationDestinationPrefixes("/app");
+        registry.enableSimpleBroker("/topic");
```

Comment:
hmm, tego nie trzeba jakos dostosowac jakos do naszej apki?

Reply:
Dobre pytanie, bo aż tak niestety nie ogarniam jeszcze websocketów. Jednak na tyle na ile testowałem to websockety automagicznie tworzą tam endpointy, na które można wysyłać wiadomości i to działa poprawnie. Jeśli dobrze rozumiem to to po prostu mówi gdzie będzie komunikacja i to tworzy te endpointy, ale mogę się mylić.